packages/protocol/src/groups/OperationalGroupKeySet.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2022-2025 Matter.js Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import type { GroupKeyManagement } from "#clusters/group-key-management";
+
+export type GroupKeySet = GroupKeyManagement.GroupKeySet;
+
+/** Enhanced structure of GroupKeySet to include operational data for easier operational processing. */
+export type OperationalGroupKeySet = GroupKeySet & {
```

Comment:
R.e. my comment in `FabricGroupsManager`, this could perhaps be upgraded to a class that supports `KeySet` functionality

Reply:
also handled